benchmarks/operator_benchmark/benchmark_runner.py

```diff
@@ -127,7 +128,12 @@ def main():
         workspace.GlobalInit(['caffe2', '--caffe2_log_level=0'])
         workspace.ClearGlobalNetObserver()
     if args.omp_num_threads:
+        # benchmark_utils.set_omp_threads sets the env variable OMP_NUM_THREADS
+        # which doesn't have any impact as C2 init logic has already been called
+        # before setting the env var.
```

Comment:
not just this, but because setting of env. variables programmatically from the process itself won't affect OpenMP in principle

Reply:
Let me update the comment.